jetson/power_logging/model/benchmark.py

```diff
@@ -108,48 +125,25 @@ def benchmark(args: argparse.Namespace) -> None:
     print(f"Warm complete in {time.perf_counter()-st:.2f} sec ...")
 
     print("Start timing using tensorrt backend ...")
-    torch.cuda.synchronize()
-    # Recorded in milliseconds
-    start_events = [torch.cuda.Event(enable_timing=True) for _ in range(args.runs)]
-    end_events = [torch.cuda.Event(enable_timing=True) for _ in range(args.runs)]
-
+    
     with torch.no_grad():
         for i in tqdm(range(args.runs)):
             # Hack for enabling profiling
             # https://github.com/pytorch/TensorRT/issues/1467
             profiling_dir = f"{args.result_dir}/{args.model}/trt_profiling"
             Path(profiling_dir).mkdir(exist_ok=True, parents=True)
-
-            # Records traces in milliseconds
-            # https://docs.nvidia.com/deeplearning/tensorrt/api/python_api/infer/Core/Profiler.html#tensorrt.Profiler
-            mod = list(model.named_children())[0][1]
-            mod.enable_profiling(profiler=CustomProfiler())
-
-            start_events[i].record()
             _ = model(input_data)
-            end_events[i].record()
```

Comment:
These still need to be recorded, we need these events in the output eventually. On CUDA these will be real values, on CPU they will be zeros (since this is how they are mocked in the cuda event wrapper). It's important that all the values that are pout to the benchmark metrics below are recorded and calculated. 

So I'd start with uncommenting values in BenchmarkMetrics on line 140, and see which of these lines need to be brought back